lnrpc/walletrpc/walletkit_server.go

```diff
@@ -1174,10 +1181,18 @@ func (w *WalletKit) FundPsbt(_ context.Context,
 	req *FundPsbtRequest) (*FundPsbtResponse, error) {
 
 	var (
-		err         error
-		feeSatPerKW chainfee.SatPerKWeight
+		err                   error
+		feeSatPerKW           chainfee.SatPerKWeight
+		coinSelectionStrategy base.CoinSelectionStrategy
 	)
 
+	coinSelectionStrategy, err = lnrpc.UnmarshallCoinSelectionStrategy(
```

Comment:
nit: can just use `:=` operator here and then move `var feeSatPerKW chainfee.SatPerKWeight` to directly before the `switch` statement.